ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java

```diff
@@ -1141,7 +1173,6 @@ public void onPress(int primaryCode) {
         if (primaryCode == KeyCodes.CTRL) {
             mControlKeyState.onPress();
             handleControl();
-            sendKeyDown(ic, 113); // KeyEvent.KEYCODE_CTRL_LEFT (API 11 and up)
```

Comment:
So, we're holding up on this event until the user is pressing another key? What happens in the case where an app does want to react to CTRL press?

Reply:
I assumed that CTRL is always used with another key. That's how CTRL is used most of the times on Windows, Linux too. Reverted the changes back.